### PR TITLE
feat: Configure analytics application registry category - EXO-61480 -Meeds-io/MIPs#37

### PR DESCRIPTION
**Pull request by sofyenne**
This change is going to add the analytics application category.

### CODE REVIEW

File: analytics-webapps/src/main/webapp/WEB-INF/conf/analytics/application-registry-configuration.xml [hunk 1]
```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<configuration
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
+        xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
+
+    <external-component-plugins>
```
Comment (azayati) [inline analytics-webapps/src/main/webapp/WEB-INF/conf/analytics/application-registry-configuration.xml]: indentation

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: analytics-webapps/src/main/webapp/WEB-INF/conf/analytics/application-registry-configuration.xml (hunk 1 above)
Comment (sofyenne) [reply analytics-webapps/src/main/webapp/WEB-INF/conf/analytics/application-registry-configuration.xml]: done !

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**